Signal/src/ViewControllers/ConversationView/Cells/OWSBubbleView.m

```diff
@@ -161,19 +161,52 @@ + (UIBezierPath *)maskPathForSize:(CGSize)size
     CGFloat bottomRounding
         = (useSmallCorners_Bottom ? kOWSMessageCellCornerRadius_Small : kOWSMessageCellCornerRadius_Large);
 
+    const CGFloat topAngle = 3.0f * M_PI / 2.0f;
+    const CGFloat rightAngle = 0.0f;
+    const CGFloat bottomAngle = M_PI / 2.0f;
+    const CGFloat leftAngle = M_PI;
```

Comment:
This is fine, but FYI CG has a bunch of pre-existing but confusingly-named constants like `M_PI_2` for these purposes.

Reply:
updated, thanks!